packages/uma/config/rules/test/test.ttl

```diff
@@ -0,0 +1,43 @@
+@prefix ex: <http://example.org/>.
```

Comment:
Is there a reason a new policy files is needed and one of the already existing policies in `uma/config/rules` can't be used? This to prevent more random policy files being added if we don't need them.

Reply:
For now, I would leave this one as it is different than the other ones.
It has different assigners, allowing us to test the `policies` endpoint.
Basically, only if you authenticate as a given (currently) WebID, then you are allowed to view your own policies (the ones you are assigned to).

Though, I think once the endpoint works and we also have a way to edit policies (create, update and delete), then we can talk about deleting this file.
Maybe an issue can be made after merging this PR to delete this file when managing policies becomes possible?